apps/website/screens/theme-generator/components/previews/DataGrid.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState } from "react";
+import { DxcContainer, DxcDataGrid } from "@dxc-technology/halstack-react";
+import Mode from "../Mode";
+import PreviewContainer from "./PreviewContainer";
+
+type DataGridPropsType = React.ComponentProps<typeof DxcDataGrid>;
+type DataGridColumnsPropsType = DataGridPropsType["columns"];
+type DataGridRowsPropsType = DataGridPropsType["rows"];
```

Comment:
This is a good warkaround, but please mark it with a // TODO for when we achieve a better strategy to handle types in the library.

Reply:
I think we are already aware so I don't think it is necessary.